### ISSUE TITLE
drifting Starter block

### ISSUE THREAD

**Issue by Tackietacktack** [tsunamayo/Starship-EVO#5433]
### Short Description of the issue:

https://github.com/tsunamayo/Starship-EVO/assets/97230091/2908fd78-83fc-4cd3-8821-11d7e93ac506

starterblock drifting away
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/13999444/Player.log)

...

### Steps to reproduce the issue:

No idea, long session

### Environment
- Starship EVO version: 24w02a
- Graphics Card: NVIDIA GeForce RTX 2060

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by tsunamayo** [comment]
@Tackietacktack Hi, do you get that often?

**Comment by Tackietacktack** [comment]
No, I had it one time. In combination with this bug #5432. Hm, can it be that planet was broken and moved the starter block, sometimes I have so what but that was for long time ago.
